PhysicsTools/PatUtils/interface/SmearedJetProducerT.h

```diff
@@ -200,6 +202,13 @@ class SmearedJetProducerT : public edm::stream::EDProducer<> {
                     resolution = JME::JetResolution::get(setup, m_jets_algo_pt);
                     resolution_sf = JME::JetResolutionScaleFactor::get(setup, m_jets_algo);
                 }
+
+                if(m_useDeterministicSeed) {
+                    unsigned int runNum_uint = static_cast <unsigned int> (event.id().run());
+                    unsigned int evNum_uint = static_cast <unsigned int> (event.id().event()); 
+                    std::uint32_t seed = runNum_uint + 4*evNum_uint;
+                    m_random_generator.seed(seed);
```

Comment:
I think that the current change introduces a fix to one problem (reproducibility/more deterministic setup), but then it also introduces a problem of repeated values where randomness is likely needed.
So, I don't think that we can defer the fix to the next PR with some precedence reference.
If JME tells me that repeated values are not a problem (perhaps with some explanation why not)  we could probably move on.